pkg/core/chain/chain.go

```diff
@@ -509,6 +521,12 @@ func (c *Chain) runStateTransition(tipBlk, blk block.Block) (*block.Block, error
 		WithField("state_hash", util.StringifyBytes(respStateHash)).WithField("e_prov", eligibleProvisioners).
 		Info("state transition completed")
 
+	provisioner, _ := base58.Encode(blk.Header.GeneratorBlsPubkey)
+	logger.WithField("provisioner", provisioner).
```

Comment:
nitpick: Should we name it  block_generator or just "generator" instead of "provisioner"?